site/en/docs/extensions/reference/commands/index.md

```diff
@@ -276,18 +276,11 @@ If an extension attempts to register a shortcut that is already used by another
 second extension's shortcut will not register as expected. You can provide a more robust end user
 experience by anticipating this possibility and checking for collisions at install time.
 
-{% Aside %}
-
-`_execute_action` will not appear in the list
-of commands returned by `commands.getAll()`.
-
-{% endAside %}
-
 {% Label %}service-worker.js:{% endLabel %}
 
 ```js
-chrome.runtime.onInstalled.addListener((reason) => {
-  if (reason === chrome.runtime.OnInstalledReason.INSTALL) {
+chrome.runtime.onInstalled.addListener(({reason}) => {
+  if (reason === 'install') {
```

Comment:
Did we intentionally move away from the enum here? Generally I think the enum is better whenever possible.

Reply:
I can never get this to work on my code `chrome.runtime.OnInstalledReason.INSTALL)`, why is it superior?

Reply:
Hmm, it should. Let me know if you want help debugging. In reality it's unlikely to change but it's generally considered better practice because if the string itself changes your code would still work, and it also says a bit more explicitly where the value came from.

Reply:
fair. But I'm leaving the brackets around reason, if not it won't work

Reply:
Of course! Thanks for spotting and fixing that.